### DISCUSSION TITLE
part-ii/2016-117/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#326] [Question comments]
Part II 2016 Probability and Measure: Paper 3, Section II, J 

https://questions.tripos.org/part-ii/2016-117/

**Comment by eha1234** [comment]
For the last part, I think this is https://math.stackexchange.com/questions/141017/lebesgue-measurable-set-that-is-not-a-borel-measurable-set　(or at least of a similar flavour)

**Reply by eha1234** [reply]
More precisely
<details><summary>Spoilers</summary>

Let A be the set constructed in part b. Then f(A) is a subset of the Cantor set, which you have (hopefully) shown is Borel, hence Lebesgue measurable, with Lebesgue measure 0. So every subset of the Cantor set is Lebesgue measurable. Now f(A) is a Lebesgue measurable set with $f^{-1}( f(A))$ not Lebesgue measurable so not Borel, but f is Borel measurable, so f(A) is not Borel either.
</details>
